### DISCUSSION TITLE
Crash with java error report

### DISCUSSION THREAD

**Discussion by francismass** [nicymike/serflings#85] [Q&A] [answered]
Hello,
I am running the stable version. 
I was playing a game on a big map and for quite a while.
It crashes now within the minute with the below popup
java.lang.ArrayIndexOutOfBoundsException: 226
	at net.simpleguide.b.a.j.d.b.a(SourceFile:395)
	at net.simpleguide.b.a.j.d.b.a(SourceFile:222)
	at net.simpleguide.b.a.j.b.c.b(SourceFile:1801)
	at net.simpleguide.b.a.a.c(SourceFile:1206)
	at net.simpleguide.b.a.p.e(SourceFile:632)
	at net.simpleguide.a.a.a.run(SourceFile:1422)
	at java.lang.Thread.run(Thread.java:750)
Attached the saved game.
Thanks
Francis
[game_crashes.zip](https://github.com/nicymike/serflings/files/10553156/game_crashes.zip)

**Comment by nicymike** [comment]
Thank you for the report. I need some time to check it because my computer is currently unavailable.

**Reply by francismass** [reply]
Thanks for that

**Reply by nicymike** [reply, marked as answer]
I checked the savegame and found an invalid player number at one of the stock inventories. I have no idea how this can happen. Did you create the game in Serflings? I ask because the global tick number is missing which should be present on a game created and played only with Serflings.

Nonetheless I fixed the invalid number. The game should now be playable again without errors.
[fixed-save.zip](https://github.com/nicymike/serflings/files/10726298/fixed-save.zip)

**Reply by francismass** [reply]
Thanks a lot! I confirm it works now.
Yes, I started this game using Ubisoft Settlers
Regards
Francis

**Reply by nicymike** [reply]
Maybe it was a bug in History Edition or the DOS version which is fixed in Serflings.
